### PR TITLE
Update SECURITY.md

### PR DESCRIPTION
**Pull request by mithindev**
Refactor security policy 

- Add instructions for reporting a vulnerability 
- Refine the secure development process
- Clarify the responsible disclosure guidelines 
- Update the versioning information to 1.0 

This commit updates the security policy to ensure that the Tenantee platform is secure and that security vulnerabilities are reported and addressed in a responsible manner.

### CODE REVIEW

**Review by mithindev — COMMENTED** [review]
Reviewed for checking!

File: SECURITY.md [hunk 1]
```diff
@@ -1,6 +1,19 @@
-# Reporting a Vulnerability
+# Security Policy
 
-Open an issue using [this link](https://github.com/zvonimirr/tenantee/issues/new?assignees=&labels=bug,security&template=bug_report.md&title=Vulnerability:%20).
-- Be as descriptive as you can.
-- Select appropriate labels.
-- Be patient.
+## Reporting a Vulnerability
+If you discover a security vulnerability, please send an email to zvonimirr@example.com to notify the Tenantee security team. The security team will work with you to acknowledge receipt of the report and to resolve the vulnerability as quickly as possible.
```
Comment (zvonimirr) [inline SECURITY.md:4]:
```suggestion
If you discover a security vulnerability, please send an email to zvonimir.rudinski@noubis.com to notify the Tenantee security team. The security team will work with you to acknowledge receipt of the report and to resolve the vulnerability as quickly as possible.
```

**Review by zvonimirr — CHANGES_REQUESTED** [review]
You can set the proper email, otherwise it looks good

**Comment by mithindev** [comment]
Hello @zvonimirr I have resolved the issue. Thank you for reviewing.

**Review by zvonimirr — APPROVED** [review]

**Comment by zvonimirr** [comment]
@allcontributors please add @mithindev for documentation

**Pull request merged**